.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,30 @@
+name: PR build workflow 
+
+on:
+    pull_request:
+        branches:
+            - master
+            
+jobs:
+    build:
+        name: build
+        runs-on: ${{ matrix.os }}
+        
+        strategy:
+            fail-fast: false
+            matrix:
+                os: [windows-latest]
+                version: [net452,netcoreapp2.0]
+                
+        steps:
+        - name: Checkout repository
+        - uses: actions/checkout@v2
+          
+        - name: Install dependencies
+          run: dotnet restore sdk/AWSXRayRecorder.sln --locked-mode
+          
+        - name: Build solution
+          run: dotnet build sdk/AWSXRayRecorder.sln --no-restore
```

Comment:
It's better for us to build package and test the dlls in `Release` configuration as it is the final stage that we [provided](https://github.com/aws/aws-xray-sdk-dotnet/blob/master/buildtools/build.proj#L7) for end users

Reply:
Yup